blog/views/admin_post_views.py

```
@@ -1,22 +1,97 @@
 from blog.services.post_create_service import create_post
 from blog.services.post_delete_service import delete_post
 from blog.services.post_content_sanitize_service import sanitize_post_content
 
+MAX_PIN_COUNT = 12
+POSTS_PER_PAGE = 10
+logger = logging.getLogger(__name__)
+
+
+def admin_access_required(view_func):
+    @wraps(view_func)
+    def _wrapped_view(request, *args, **kwargs):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_wrapped_view`

(ANN202)